app/models/grda_warehouse/service_history_enrollment.rb

```diff
@@ -241,6 +241,10 @@ def self.service_types
 
   end
 
+  scope :heads_of_households, -> () do
+    where(she_t[:head_of_household].eq(true))
```

Comment:
this can just be `where head_of_household: true` no need for arel.

Reply:
hmm, this appears necessary for the merge to work.

Reply:
There is a `has_one :head_of_household` which I believe overloads the column name, hence the Arel.